utility/BUILD.bazel

```diff
@@ -17,6 +17,22 @@ MIGRATED_PROD_FILES = glob([
     "src/main/java/org/oppia/android/util/system/*.kt",
 ])
 
+filegroup(
+    name = "manifest",
```

Comment:
Is this still needed? Suggest removing it.

Reply:
Done